deploy.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+
+setup_git() {
+  git config --global user.email "travis@travis-ci.org"
+  git config --global user.name "Travis CI"
+}
+
+commit_files() {
+  git checkout -b build-branch
+  git pull
+  git add .
```

Comment:
Do we need to add all files? Can we use a regex or expand gitignore to ignore potentially unnecessary files being added?

Reply:
Good catch. Yeah, I need to add the `vendor/` directory to the gitignore from what I can tell. All of that stuff seems to be added to `vendor/`